agent/lib_aws_sdk_php.h

```diff
@@ -87,7 +110,9 @@ extern void nr_lib_aws_sdk_php_sqs_handle(nr_segment_t* segment,
  *
  * Note: The caller is responsible for freeing cloud_attrs->cloud_resource_id
  */
-void nr_aws_sdk_lambda_client_invoke_parse_args(NR_EXECUTE_PROTO, nr_segment_cloud_attrs_t* cloud_attrs);
+void nr_aws_sdk_lambda_client_invoke_parse_args(
+    NR_EXECUTE_PROTO,
+    nr_segment_cloud_attrs_t* cloud_attrs);
```

Comment:
This appears to be a formatting only change.

Reply:
yes.